django/db/backends/base/base.py

```diff
@@ -238,11 +241,15 @@ def _cursor(self, name=None):
 
     def _commit(self):
         if self.connection is not None:
+            if self.queries_logged:
+                logger.debug("COMMIT")
```

Comment:
This should be moved to the `CursorDebugWrapper`, e.g.
```python
def commit(self):
    with self.debug_sql(sql='COMMIT'):
        return super().commit()
```

Reply:
I may be missing something, but `commit` is not a method on the cursor, but on the connection. And the connection comes from `get_new_connection` from the individual backends.

Reply:
Right.